src/apps/filters/h3.c

```diff
@@ -309,11 +309,11 @@ SUBCOMMAND(getIndexDigit,
     Arg *args[] = {&getIndexDigitArg, &helpArg, &cellArg, &digitArg};
     PARSE_SUBCOMMAND(argc, argv, args);
     // TODO: Should there be a general `isValidIndex`?
-    H3Error cellErr = H3_EXPORT(isValidCell)(cell);
-    H3Error edgeErr = H3_EXPORT(isValidDirectedEdge)(cell);
-    H3Error vertErr = H3_EXPORT(isValidVertex)(cell);
-    if (cellErr && edgeErr && vertErr) {
-        return cellErr;
+    bool isValid = H3_EXPORT(isValidCell)(cell) ||
```

Comment:
I see how I made this mistake. In most of the code the error condition is the "truthy" value, but here success is `true`, though I named these variables as if they were `H3Error`s. That feels like a point against [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation).